google-cloud-datastore/src/test/java/com/google/cloud/datastore/it/ITDatastoreTest.java

```diff
@@ -729,4 +730,46 @@ public void testDelete() {
     assertNull(keys.next());
     assertFalse(keys.hasNext());
   }
+
+  @Test
+  public void testRunInTransaction() {
+    Datastore.TransactionCallable<Integer> callable1 =
+        new Datastore.TransactionCallable<Integer>() {
+          private Integer attempts = 0;
+
+          public Integer run(DatastoreReaderWriter transaction) {
+            transaction.get(KEY1);
+            if (attempts < 1) {
```

Comment:
Subjective. Please consider a shorter form, for example:
`return attempts++ < 1 :  DatastoreException(10, "", "ABORTED", false, null) : attempts;`
And then change from `assertEquals(result, 1)` to `assertEquals(result, 2)`, as 2 is the actual number of attempts here (first failed, second succeded)).